bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/libmatomo.sh

```diff
@@ -204,6 +204,15 @@ EOF
             ini-file set -s "mail" -k "encryption" -v "$MATOMO_SMTP_PROTOCOL" "$MATOMO_CONF_FILE"
         fi
 
+        # Optional noreply name and address
+        if ! is_empty_value "$MATOMO_NOREPLY_NAME"; then
+            ini-file set -s "General" -k "noreply_email_name" -v "$MATOMO_NOREPLY_NAME" "$MATOMO_CONF_FILE"
+        fi
+
+        if ! is_empty_value "$MATOMO_NOREPLY_ADDRESS"; then
+            ini-file set -s "General" -k "noreply_email_address" -v "$MATOMO_NOREPLY_ADDRESS" "$MATOMO_CONF_FILE"
+        fi
```

Comment:
Does it make sense to add these `ini-file set` inside the `if` block above? The one for the SMTP configuration. Do the `noreply_email_name` and `noreply_email_address` make any effect if the SMTP configuration is not set?

Reply:
I have tested the matomo configuration inside the current matomo container and you are right. The `noreply_email_name` and `noreply_email_address` won't have any effect since the default way for matomo to send mails is sendmail which doen't exist in this container anyway.
So I have made the change and updated the pr

Reply:
Thank you! We will do some additional testing and I might push additional changes to your branch regarding the environment variables. After that, the PR should be good to go.